<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>234</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="2">
     <widget class="QPushButton" name="startFunction">
      <property name="text">
       <string>run function</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="confirmName">
      <property name="text">
       <string>confirm name </string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="writeCom">
      <property name="text">
       <string>Write Comport</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>com25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>com5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>com3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>com10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>com7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>com11</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>enter name of tex document to save data</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>select sensortype</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QComboBox" name="SensorSelect">
      <item>
       <property name="text">
        <string>Force Pressure / Piezo Sensor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MPU6050</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>372</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
